examples/react-es6/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "react": "0.13.0-beta.1",
+    "react-tools": "0.13.0-beta.1"
+  },
+  "devDependencies": {
+    "babel-jest": "*"
```

Comment:
I could be mistaken but I was under the impression that `import` statements weren't supported with the harmony transformer.